wp/lib/bap_wp/src/precondition.ml

```diff
@@ -728,51 +728,54 @@ let mk_env
   Env.mk_env ~subs ~specs ~default_spec ~indirect_spec ~jmp_spec ~int_spec ~exp_conds ~num_loop_unroll
     ~arch ~freshen_vars ~use_fun_input_regs ~stack_range ~data_section_range ctx var_gen
 
+let conditional_jmp (jmp : Jmp.t) (env : Env.t) (target_pre : Constr.t)
```

Comment:
In my opinion, it would a good idea to add a description of `conditional_jmp` to `precondition.mli` as well.
Any thoughts?

Reply:
I agree, I should document this function. There's nothing written about this right now.

Reply:
I added documentation for `conditional_jmp` in `precondition.ml`. I did not want to expose this function in the `mli`, since it's a helper for `visit_jmp` and shouldn't really be called on its own.

Reply:
That sounds like a better idea.